classes/TaskRunner/Miscellaneous/UpdateConfig.php

```diff
@@ -32,61 +32,74 @@
  */
 class UpdateConfig extends AbstractTask
 {
+    /**
+     * @var bool
+     */
+    protected $isRunInCLI = false;
+
+    /**
+     * Data being passed by CLI entry point
+     *
+     * @var array
+     */
+    protected $cliParameters;
+
     public function run()
     {
         // nothing next
         $this->next = '';
 
         // Was coming from AdminSelfUpgrade::currentParams before
-        $request = $this->getRequestParams();
+        $inputConfigurationData = $this->fetchConfigurationData();
+
         $config = array();
         // update channel
-        if (isset($request['channel'])) {
-            $config['channel'] = $request['channel'];
+        if (isset($inputConfigurationData['channel'])) {
```

Comment:
Consider using a shorter variable name like `$input` or `$params`

Reply:
Fixed in https://github.com/PrestaShop/autoupgrade/pull/359/commits/46d1d8e3942284bbf2b1fe15b7bd15976ab4d3da